diagnosis/README.md

```diff
@@ -18,7 +22,12 @@ These are the logs retrieved by the script:
 - Kubelet service status and journal
 - Etcd service status and journal
 - Gallery item's DVM logs
+- Cluster Snapshot
```

Comment:
`Microsoft Azure Linux Agent (waagent) logs` => do not think we collect these
`Cloud-init logs` => do not think we collect these
`Running kube-system containers metadata` => instead of Running Docker container metadata
`Running kube-system containers logs` => Running Docker container logs
`kube-system snapshot` => Cluster Snapshot